pylint/checkers/imports.py

```diff
@@ -277,6 +283,7 @@ def _make_graph(filename, dep_info, sect, gtype):
 
 DEFAULT_STANDARD_LIBRARY = ()
 DEFAULT_KNOWN_THIRD_PARTY = ("enchant",)
+DEFAULT_PREFERRED_MODULES = ("xml:defusedxml", "lxml:defusedxml")
```

Comment:
I don't think we should be recommending anything at this point and should be let at the latitude of each project using `pylint` to define their own set of preferred modules.

Reply:
Sure thing, I changed that in 2dc8e3a